docs/TestingAsyncCode.md

```diff
@@ -21,7 +21,7 @@ test('the data is peanut butter', () => {
 
 ## Async/Await
 
-Alternatively, you can use `async` and `await` in your tests. To write an async test, use the `async` keyword in front of the function passed to `test`. For example, the same `fetchData` scenario can be tested with:
+If you expect a promise to be rejected, use the `.catch` method. Make sure to add `expect.assertions` to verify that a certain number of [assertions](ExpectAPI.md#assertion-count) are called. Otherwise, a fulfilled promise would not fail the test.
```

Comment:
I don't think it makes sense to recommend `.catch` here, as we use `catch` explicitly